src/options.rs

```diff
@@ -732,6 +734,26 @@ pub fn get_app_use_regex(matches: &ArgMatches, config: &Config) -> bool {
     false
 }
 
+pub fn get_use_terminal_name(matches: &ArgMatches, config: &Config) -> (bool, String) {
+    if matches.is_present("title") {
+        return if let Some(custom_name) = matches.value_of("title") {
+            (true, String::from(custom_name))
+        } else {
+            (false, String::from(""))
+        };
+    } else if let Some(flags) = &config.flags {
+        if let Some(title_to_hostname) = flags.title_to_hostname {
+            if title_to_hostname {
+                return match gethostname().into_string() {
+                    Ok(hostname) => (title_to_hostname, format!("btm ({})", hostname)),
+                    Err(_) => (false, String::from("")),
+                };
+            }
+        }
+    }
+    (false, String::from(""))
+}
+
```

Comment:
Some suggestions:

- As mentioned above, remove the custom name part, as we're not doing that for now. Just use the hostname.
- Why not just return an `Option`? e.g. if there is no change, return `None`.